chain/chunks/src/logic.rs

```diff
@@ -46,6 +48,31 @@ pub fn cares_about_shard_this_or_next_epoch(
         || shard_tracker.will_care_about_shard(account_id, parent_hash, shard_id, is_me)
 }
 
+pub fn get_shards_cares_about_this_or_next_epoch(
```

Comment:
Should we have this return a list of shard UIDs? I see that you're converting them to UIDs afterwards anyway and it seems more correct w.r.t. resharding?

Reply:
Ah I see this was discussed before. This is not important for now.